interpreter/binary/decode.ml

```diff
@@ -190,6 +190,7 @@ let heap_type s =
       | -0x15 -> StructHT
       | -0x16 -> ArrayHT
       | -0x17 -> ContHT
+      | -0x18 -> NoContHT
```

Comment:
Can we order this next to the other bottom types, i.e., -0x0c?

Reply:
Done. I changed the opcode for `ContHT` to `-0x18` as the other code appears to have been snatched by the exception handling proposal already.

Reply:
Ah, FWIW, that will need a bottom type as well when combined with the GC proposal, so perhaps better even use -0x0b for nocont?

Reply:
good idea